spec/javascripts/models/beta/story_spec.js

```diff
@@ -862,4 +862,19 @@ describe('Story model', function () {
       expect(Story.cloneStory(story).documents).toEqual([]);
     });
   });
+
+  describe("possibleStatesFor", () => {
+    it("should return all states", () => {
```

Comment:
Can we add a nested for each of these cases to explain why they return it? Like this:

```js
describe('when state is started', () => {
  it('returns all states', () => { /* ... */ });
});
```

and then do the same for the test below.